src/components/activity-dialog/activity-dialog.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { createRef, useEffect } from "react";
 import { dialogStyles } from "./activity-dialog.styles";
-import { Text, Headline, Button } from "@fluentui/react-components";
+import { Text, Button, Title1 } from "@fluentui/react-components";
```

Comment:
Had to remove `Headline` and replace with `Title1` as `Headline` is not present in latest @fluent/react-component package